src/Orchard.Cms.Web/Modules/Orchard.Roles/Services/RoleStore.cs

```diff
@@ -58,10 +58,10 @@ public void UpdateRoles(RolesDocument roles)
             _memoryCache.Set(Key, roles);
         }
 
-        public async Task<IEnumerable<string>> GetRoleNamesAsync()
+        public async Task<IEnumerable<RoleNamesEntry>> GetRoleNamesAsync()
```

Comment:
What if you need to "display" something else next?

Can you remind me where `IRoleProvider` is used, and if it's our code or AspNet one?